.env.sample

```diff
@@ -3,3 +3,4 @@ DEVISE_KEY=
 API_URL=
 PORT=
 RAILS_ENV=
+APIGATEWAY_URL=http://..
```

Comment:
we already have an "API_URL" param. either reuse it or rename both to make it clear what's the difference. Us-in-6-months will not remember what the difference is.
Also a good idea is adding these vars to the README and document what they do - see https://github.com/Vizzuality/GlobalFishingWatch as an example